deployments/external/docker-compose.yml

```diff
@@ -64,21 +64,23 @@ services:
     ports:
     - 8080:8080
     environment:
-      KC_HOSTNAME_URL: 'http://login.example.com'
```

Comment:
I updated the environment variables to the `hostnames_v2` feature which is now enabled by default.